frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -35,13 +36,17 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({ state }) => {
     securityAndNetwork,
     createLocalVolumeSet,
     backingStorage,
+    connectionDetails,
+    createStorageClass,
     nodes,
   } = state;
   const { capacity, arbiterLocation, enableArbiter } = capacityAndNodes;
   const { encryption, kms, networkType } = securityAndNetwork;
   const { deployment, externalStorage, type } = backingStorage;
 
   const isMCG = deployment === DeploymentType.MCG;
+  const isRhcs = !_.isEmpty(connectionDetails);
+  const isStandaloneExternal = hasOCS && !_.isEmpty(createStorageClass);
```

Comment:
One doubt... we also have already set flags to check whether it is external cluster or not, will that not work here ??

Reply:
If independent mode is true then we can say its rhcs but if its false we cannot say -
1. a non rhcs cluster created or
2. a non rhcs cluster not present

Also in future we will have many storage systems, so keeping this way, would not complicate